components/src/css/sprinkles.css.ts

```diff
@@ -208,8 +208,15 @@ const selectorProperties = defineProperties({
         vars: { [boxShadowColorVar]: vars.colors.foregroundSecondary },
         boxShadow: `${vars.shadows['0']} ${boxShadowColorVar}`,
       },
+      '-1px': {
```

Comment:
would change this to `-px` so it follows convention.

Reply:
ah good one, will do